sleap/gui/app.py

```diff
@@ -1391,6 +1391,10 @@ def updateStatusMessage(self, message: Optional[str] = None):
             else:
                 self.statusBar().setStyleSheet("color: black")
 
+            # TODO(JS): Add Camera indicator test here
```

Comment:
We normally add tests in the tests folder, but for this particular PR (since it is extremely visual/GUI-based) I would just manually test it by using the GUI with

```bash
sleap-label tests\data\cameras\minimal_session\min_session_frame_groups.slp
```

and then clicking through the Videos to see that the `statusBar` is updated accordingly.